src/Packetery/Module/Order/Builder.php

```diff
@@ -132,17 +132,7 @@ public function finalize( WC_Order $wcOrder, Entity\Order $order ): Entity\Order
 
 		$order->setSize( $order->getSize() );
 
-		if ( $order->isExternalCarrier() ) {
-			$carrier = $this->carrierRepository->getById( (int) $order->getCarrierId() );
-			$order->setCarrier( $carrier );
-			$order->setCarrierCode( $order->getCarrierId() );
-		} else {
-			$order->setCarrierCode(
-				$this->pickupPointsConfig->getCompoundCarrierIdByCountry(
-					$order->getShippingCountry()
-				)
-			);
-		}
+		$order->setCarrierCode( $this->getCarrierId( $order ) );
```

Comment:
Nevznikl carrierCode proto, aby obsahoval přetransformované carrierId ? CarrierId může obsahovat "packeta". CarrierCode obsahoval třeba zpointcz a nebo 106.

Mně se nelíbí to, že se do carrierCode najednou cpe "packeta". Dával bych tam prostě NULL. A na relevantních místech kontroloval NULL.

To znamená že by setCarrierCode() přijímal NULL.